apps/server-asset-sg/docker/Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV NODE_ENV=development
 WORKDIR /app
 COPY . .
 
-RUN npm install && ./node_modules/.bin/prisma generate

```

Comment:
Why is this no longer needed?

Reply:
This is also called in `RUN npx nx build server-asset-sg --configuration=production` which happens in the next line, see https://github.com/swisstopo/swissgeol-assets-suite/blob/ef5764841053e4f4f6d7f464acd7e27ccce9e234/apps/server-asset-sg/project.json#L14